helidon-archetype/engine/src/test/java/io/helidon/build/archetype/engine/ArchetypeBaseTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.build.archetype.engine;
+
+import java.io.File;
+import java.net.URL;
+
+import org.junit.jupiter.api.BeforeAll;
+
+/**
+ * Class ArchetypeBaseTest.
+ */
+public class ArchetypeBaseTest {
+
+    static private File targetDir;
+
+    @BeforeAll
+    static void initialize() {
+        URL url = ArchetypeEngineTest.class.getClassLoader().getResource("test-se.jar");
```

Comment:
Where does this jar come from?

Reply:
It is a sample jar I checked in as a resource to test this.